obelisk_ws/src/obelisk_ros/config/dummy_cpp.yaml

```diff
@@ -82,6 +82,20 @@ onboard:
           sensor_settings:
           - topic: /obelisk/dummy/sensor
             dt: 0.001
-            sensor_type: jointpos
+            sensor_type: ObkJointEncoders
```

Comment:
if we're going to do this, let's rename the `sensor_type` field to `msg_type` to be unambiguous